src/SessionSetup/RetrieveInstancesWidgetTest.cpp

```diff
@@ -84,8 +82,20 @@ const Instance kTestInstance2{
     "IN_USE",                                                   /* state */
 };
 
+const Instance kTestInstance3{
+    "Test Instance 3",                                          /* display_name */
+    "test_instance_3_id",                                       /* id */
```

Comment:
Nit, I would generally recommend to mimic the name of a real instance ID, especially since those contain forward slashes that may cause issues in certain scenarios (even though it's highly unlikely this is the case in this test).

Reply:
done